tests/integration/solidity_contracts/PlainOpcodes/PlainOpcodes.t.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity 0.8.10;
```

Comment:
Is this test file run against forge and then we compare results with kakarot somewhere?

would be good to have a top level comment contextualizing it.

Reply:
yes it's run against `forge`, I wrote this test to debug my `opcodeExtCodeCopyReturnsCounterCode`.

The idea was that we write solidity to test kakarot, but if this solidity is not trivial, how can we debug it? Since we use forge to compile, I guesses that the easiest way was to write a forge test...
This is not run in the CI, was just for debug purpose, but left it here because:
- I expect this (testing the solidity before testing it with kakarot) to be a common need
- we may use `forge test` and diff testing for kakarot

Reply:
I'm updating the README#test section instead to detail this usecase